Source/Core/DolphinQt/ResourcePackManager.cpp

```diff
@@ -167,10 +167,15 @@ void ResourcePackManager::Change()
   if (items.empty())
     return;
 
-  if (ResourcePack::IsInstalled(ResourcePack::GetPacks()[items[0]->row()]))
+  if (ResourcePack::IsInstalled(
+          ResourcePack::GetPacks()[m_table_widget->rowCount() - 1 - items[0]->row()]))
```

Comment:
I wonder if `m_table_widget->rowCount() - 1 - items[0]->row()` should be a named local. Or at least commented, since I can't tell what it does (or rather, why).